railties/lib/rails/generators/rails/app/templates/config/initializers/feature_policy.rb.tt

```diff
@@ -0,0 +1,11 @@
+# Define an application-wide HTTP feature policy. For further
+# information see https://developers.google.com/web/updates/2018/06/feature-policy
+#
+# Rails.application.config.feature_policy do |f|
+#   f.camera      :none
+#   f.gyroscope   :none
+#   f.microphone  :none
+#   f.usb         :none
+#   f.fullscreen  :self
+#   f.payment     :self, "https://secure-example.com"
```

Comment:
Unlike `example.com`, `secure-example.com` is a domain that can be used normally. It should not be used as an example.

Reply:
Good catch. While this is only an example for configuration, we should definitely avoid using real world domains and instead restrict to using using the [reserved domains](https://tools.ietf.org/html/rfc2606#section-3). I'll open a PR to address.